config/routes.rb

```diff
@@ -1,5 +1,5 @@
 Rails.application.routes.draw do
-  resources :users
-  post '/users', to: 'users#create'
+  resources :users 
```

Comment:
não precisa usar o `resources`. 
ou se for utiliza-lo, limite apenas ao post.

Reply:
Entendi